### PR TITLE
v2.1, v3.0, dev: add a warning in batch operations

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 

- Add a warning concerning the transaction in data loading

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1586

### Which version does your change affect? 

v2.1, v3.0, dev

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by lilin90** [comment]
@TomShawn Please help review this PR. Thanks!

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 1]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - Scope: SESSION
 - Default value: 0
 - This variable is used to set whether to divide the inserted data automatically. It is valid only when `autocommit` is enabled.
-- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When inserting a large amount of data, you can set the variable value to `1`. Then, the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 2]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
-- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When deleting a large amount of data, you can set the variable value to `1`. Then, the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: dev/reference/sql/statements/load-data.md [hunk 3]
```diff
@@ -41,6 +41,10 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * TiDB will by default commit every 20 000 rows. This behavior is similar to MySQL NDB Cluster, but not the default configuration with the InnoDB storage engine.
 
+> **Note:**
+> 
+> Committing through splitting transactions is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline dev/reference/sql/statements/load-data.md]:
```suggestion
> Committing through splitting a transaction is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: dev/reference/transactions/transaction-model.md [hunk 4]
```diff
@@ -75,4 +75,4 @@ Due to its distributed nature, workloads that are single-threaded or latency-sen
 
     > **Warning:**
     >
-    > The `LOAD DATA` operation in TiDB does not guarantee the atomicity of transactions, so it is recommended only for initial batch loading data, and not during regular usage for production environment(s).
\ No newline at end of file
+    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, it is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline dev/reference/transactions/transaction-model.md]:
```suggestion
    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, this operation is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v2.1/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 5]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - Scope: SESSION
 - Default value: 0
 - This variable is used to set whether to divide the inserted data automatically. It is valid only when `autocommit` is enabled.
-- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v2.1/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When inserting a large amount of data, you can set the variable value to `1`. Then, the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v2.1/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 6]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
-- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v2.1/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When deleting a large amount of data, you can set the variable value to `1`. Then, the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v2.1/reference/sql/statements/load-data.md [hunk 7]
```diff
@@ -41,6 +41,10 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * TiDB will by default commit every 20 000 rows. This behavior is similar to MySQL NDB Cluster, but not the default configuration with the InnoDB storage engine.
 
+> **Note:**
+> 
+> Committing through splitting transactions is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v2.1/reference/sql/statements/load-data.md]:
```suggestion
> Committing through splitting a transaction is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v3.0/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 8]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - Scope: SESSION
 - Default value: 0
 - This variable is used to set whether to divide the inserted data automatically. It is valid only when `autocommit` is enabled.
-- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When inserting a large amount of data, you can set the variable value to 1. Then the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v3.0/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When inserting a large amount of data, you can set the variable value to `1`. Then, the inserted data is automatically divided into multiple batches and each batch is inserted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v3.0/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 9]
```diff
@@ -177,16 +177,14 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
-- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction.
-- This operation might lead to a loss of transaction atomicity. Therefore, it is not recommended to use this parameter in the production environment.
+- When deleting a large amount of data, you can set the variable value to 1. Then the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v3.0/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
- When deleting a large amount of data, you can set the variable value to `1`. Then, the data for deletion is automatically divided into multiple batches and each batch is deleted by a single transaction. This operation breaks the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v3.0/reference/sql/statements/load-data.md [hunk 10]
```diff
@@ -41,8 +41,12 @@ Records: 815264  Deleted: 0  Skipped: 0  Warnings: 0
 
 * TiDB will by default commit every 20 000 rows. This behavior is similar to MySQL NDB Cluster, but not the default configuration with the InnoDB storage engine.
 
+> **Note:**
+> 
+> Committing through splitting transactions is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v3.0/reference/sql/statements/load-data.md]:
```suggestion
> Committing through splitting a transaction is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v3.0/reference/transactions/transaction-model.md [hunk 11]
```diff
@@ -76,4 +76,4 @@ Due to its distributed nature, workloads that are single-threaded or latency-sen
 
     > **Warning:**
     >
-    > The `LOAD DATA` operation in TiDB does not guarantee the atomicity of transactions, so it is recommended only for initial batch loading data, and not during regular usage for production environment(s).
+    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, it is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v3.0/reference/transactions/transaction-model.md]:
```suggestion
    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, this operation is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

File: v2.1/reference/transactions/transaction-model.md [hunk 12]
```diff
@@ -75,4 +75,4 @@ Due to its distributed nature, workloads that are single-threaded or latency-sen
 
     > **Warning:**
     >
-    > The `LOAD DATA` operation in TiDB does not guarantee the atomicity of transactions, so it is recommended only for initial batch loading data, and not during regular usage for production environment(s).
+    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, it is at the expense of breaking the atomicity and isolation of the transaction. When using it, you need to ensure that there are **not any other** ongoing operations on the table, and in the event of an error, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended for use in a production environment.
```
Comment (TomShawn) [inline v2.1/reference/transactions/transaction-model.md]:
```suggestion
    > The `LOAD DATA` operation in TiDB by default splits transactions and commits them in batches. However, this operation is at the expense of breaking the atomicity and isolation of the transaction. When performing this operation, you must ensure that there are **no other** ongoing operations on the table. When an error occurs, **manual intervention is required to check the consistency and integrity of the data**. Therefore, it is not recommended to set this parameter in a production environment.
```

**Pull request merged**